src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -217,7 +219,7 @@
   </Target>
 
   <!-- archive the test binaries along with some supporting files -->
-  <Target Name="ArchiveTestBuild" AfterTargets="CreateAssemblyListTxt" Condition="'$(ArchiveTests)' == 'true'">
+  <Target Name="ArchiveTestBuild" AfterTargets="CreateAssemblyListTxt" Condition="'$(ArchiveTests)' == 'true'"  DependsOnTargets="RunTestsForProject">
```

Comment:
I assume this target will get some more work done to it when we get closer to getting finished.